### DISCUSSION TITLE
San-Serif font on docs website

### DISCUSSION THREAD

**Discussion by agosmou** [pallets/flask#5162] [Ideas]
Thoughts on having the flask docs website change to a sans-serif font like arial?
I feel it's easier to read and navigate on a screen (see attachments).
***
Proposed Font
![image](https://github.com/pallets/flask/assets/73868258/28e34d76-b860-4936-b4fa-e207238024c9)

Current Font
![image](https://github.com/pallets/flask/assets/73868258/4b69c2c2-c973-4b87-8da1-9cb379eb23e0)

**Comment by abhint** [comment]
@agosmou If you want to change something, create PR. 😄

**Comment by davidism** [comment]
I don't plan to change this. I feel that sans-serif is easy to read and navigate on screen :wink:

**Comment by davebarkerxyz** [comment]
I've also noticed how illegible the default font stack is on latest Chrome under Windows 11 (4K screen at 150% scaling). The default font face / size don't render particularly well in that configuration. However the docs site is perfectly readable (and aesthetically pleasing) on Mac OS Ventura 13.4.1 with Safari 16.5.1.

The stack appears to be `'Garamond', 'Georgia', serif`. I suspect Windows, with the default installed fonts, is using Georgia.

**Comment by davebarkerxyz** [comment]
To make my comment more productive - @davidism you say you don't plan to change this, but would you be open to a PR that changes Georgia to something that renders better in Chrome on Windows (or dropping it in favour of just `'Garamond', serif` if the default fallback font is usable) if I test some combinations on Windows? I agree with @agosmou, but also concede that switching for another serif font would be less of a dramatic change.

It looks like the issue (rendering Georgia in the browser on Windows) goes back over 10 years https://stackoverflow.com/questions/9160021/georgia-font-does-not-render-correctly-on-windows

**Comment by davebarkerxyz** [comment]
Apologies for re-commenting on this thread (which I’ve only noticed now is closed) but I’ve done some more testing and found that the issue seems to be with Garamond rendering under Chrome on Windows 11 (not with Georgia as originally suspected). With Garamond in the stack, it renders like:

<img width="938" alt="flask-docs-windows" src="https://github.com/pallets/flask/assets/5273579/9f05526c-fc35-4fa0-a9af-4a2c3f563c15">

Removing Garamond and using either Georgia or the fallback `serif` renders more consistently compared to Mac OS.

I know this isn’t a Flask (or Flask Sphinx template) issue - just highlighting how poorly Chrome renders the default stylesheet in Windows 11.
